components/dashboard/ee/src/components/admin/workspace-view.tsx

```diff
@@ -95,7 +95,7 @@ export class WorkspaceView extends React.Component<WorkspaceViewProps, Workspace
             <Grid container>
                 <Grid item xs={8}><Typography variant="h2">Workspace: {this.props.workspaceID}</Typography></Grid>
                 <Grid item xs={4} style={{textAlign: "right"}}>
-                    <Button href={`/api/enforcement/kill-workspace/${this.props.workspaceID}`} variant="contained">Kill</Button>
+                    <Button href={`/api/enforcement/kill-workspace/${this.props.workspaceID}`} variant="contained">Stop Immediately</Button>
```

Comment:
**thought**: Didn't get this in time but adding here some thoughts. Renaming this to _Stop_ instead of _Stop Immediately_ could be better but we'll get back to this once the dashboard redesign starts going through the admin dashboard. /cc @corneliusludmann @csweichel